<launch>
	<include file="$(find lasr_perception_server)/launch/perception_server.launch"/>
	<rosparam command="load" file="$(find coffee_shop)/config/test_check_table.yaml" />
	<rosparam command="load" file="$(find coffee_shop)/config/motions.yaml" />
	<node pkg="pcl_segmentation" type="pcl_segmentation_server" name="pcl_segmentation_server" output="screen" />
	<node pkg="coffee_shop" type="test_check_table.py" name="test_check_table" output="screen"/>
</launch>
